src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSourceRequest.java

```diff
@@ -321,25 +333,27 @@ public final void setTags(@CheckForNull Iterable<Tag> tags) {
         return Util.fixNull(tags);
     }
 
-
-    // TODO Iterable<Tag> getTags() and setTags(...)
-
     /**
-     * Returns the names of the project collaborators or {@code null} if those details have not been provided yet.
+     * Returns the project {@link Member} or {@code null} if those details have not been provided yet.
      *
-     * @return the names of the project collaborators or {@code null} if those details have not been provided yet.
+     * @return the project {@link Member} or {@code null} if those details have not been provided yet.
      */
-    public final Set<String> getCollaboratorNames() {
-        return collaboratorNames;
+    public final HashMap<String, AccessLevel> getMembers() {
+        return members;
     }
 
     /**
-     * Provides the request with the names of the project collaborators.
+     * Provides the set of project {@link Member}.
      *
-     * @param collaboratorNames the names of the project collaborators.
+     * @param members the set of project {@link Member}.
      */
-    public final void setCollaboratorNames(@CheckForNull Set<String> collaboratorNames) {
-        this.collaboratorNames = collaboratorNames;
+    public final void setMembers(@CheckForNull List<Member> members) {
+        this.members = new HashMap<>();
+        if(members != null) {
```

Comment:
You could also skip this if you move it up
```suggestion
```

Reply:
This checks the function parameter variable and not the class member variable.

Reply:
Fixed in 60467ab